atlite/datasets/era5.py

```diff
@@ -290,9 +292,9 @@ def retrieval_times(coords, static=False, monthly_requests=False):
     time = coords["time"].to_index()
     if static:
         return {
-            "year": str(time[0].year),
-            "month": str(time[0].month),
-            "day": str(time[0].day),
+            "year": [str(time[0].year)],
+            "month": [str(time[0].month).zfill(2)],
```

Comment:
If leading zeros are required, perhaps this is better (and "%d", "%Y" for day and year):
```suggestion
            "month": [time[0].strftime("%m")],
```